homeassistant/helpers/config_validation.py

```diff
@@ -973,6 +973,56 @@ def custom_serializer(schema: Any) -> Any:
     return voluptuous_serialize.UNSUPPORTED
 
 
+def expand_condition_shorthand(value: Any | None) -> Any:
+    """Expand boolean condition shorthand notations."""
+
+    if not isinstance(value, dict):
```

Comment:
```suggestion
    if not isinstance(value, dict) or CONF_CONDITIONS in value:
```

Reply:
Stupid misses like this is why I really appreciate code reviews.